sklearn/tree/_criterion.pyx

```diff
@@ -962,6 +969,304 @@ cdef class MSE(RegressionCriterion):
         impurity_left[0] /= self.n_outputs
         impurity_right[0] /= self.n_outputs
 
+cdef class MAE(RegressionCriterion):
```

Comment:
hmm I'm not sure what you mean here; we did override node_value to predict the median? https://github.com/scikit-learn/scikit-learn/pull/6667/files#diff-0bee2b90794ea41597c4c4e3c6b6388eR1184

If you meant something else, could you explain a bit further?